### PR TITLE
ADD module product_variant_update_price

### PR DESCRIPTION
**Pull request by bealdav**
make it easy to increase / decrease price of variants in a massive way.

### CODE REVIEW

File: product_variant_update_price/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+This module make it easy to increase / decrease price of variants in a massive way.
+
+
+Usage
+=====
+
+A wizard allows to apply a percentage to selected products
+
+
+.. figure:: static/dascription/upd_price.png
```
Comment (legalsylvain) [inline product_variant_update_price/README.rst]: description

**Review by legalsylvain — COMMENTED** [review]
LGTM. Code review. No test.

File: product_variant_update_price/README.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==============================
+Product Variants Update Prices
+==============================
+
+This module make it easy to increase / decrease price of variants in a massive way.
```
Comment (rvalyi) [inline product_variant_update_price/README.rst]: make -> makes

File: product_variant_update_price/wizards/update_price.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+from openerp import models, api, fields
+
+
+class ProductUpdatePrice(models.TransientModel):
+    _name = "product.update.price"
+    _description = "Allow to update sale price"
+
+    percentage = fields.Float(
+        required=True,
+        help="Percentage value to increase sale price")
```
Comment (rvalyi) [inline product_variant_update_price/wizards/update_price.py]: Sale price percentage increase

**Review by rvalyi — CHANGES_REQUESTED** [review]
small English details to fix

**Review by hparfr — APPROVED** [review]

**Review by rvalyi — APPROVED** [review]

**Pull request merged**